### ISSUE TITLE
Create a model to detect inactive bugs

### ISSUE THREAD

**Issue by marco-c** [mozilla/bugbug#1267]
This is not so easy, as it first requires to have a clear definition of what an inactive bug is.

**Comment by ayush-1506** [comment]
By definition, aren't stale bugs the ones where there hasn't been any activity for a while? Or can new bugs also be classified as stale?

**Comment by marco-c** [comment]
> By definition, aren't stale bugs the ones where there hasn't been any activity for a while?

Not exactly, what we want is bugs where there hasn't been activity for a while (but we need to define what "a while" is exactly) and that aren't real bugs just waiting for somebody to jump in and fix them.
Basically, we want to detect bugs where there is not enough information to proceed and the reporter has been asked to provide, but has not provided for a long time, the required information to proceed.
